components/org.wso2.carbon.config/src/test/java/org/wso2/carbon/config/configprovider/ConfigProviderImplTest.java

```diff
@@ -93,14 +93,14 @@ public void yamlFileConfigObjectTestCase() throws IOException {
             Assert.assertEquals(transport2.get("name"), "pqr");
             Assert.assertEquals(transport2.get("port"), "${env:pqr.http.port}");
             Assert.assertEquals(transport2.get("secure"), "${sys:pqr.secure}");
-            Assert.assertEquals(transport2.get("desc"),
-                    "This transport will use ${env:pqr.http.port} as its port. Secure - ${sys:pqr.secure}");
+            Assert.assertEquals(transport2.get("desc"), "This transport will use ${env:pqr.http.port} as its port. " +
+                    "Secure - ${sys:pqr.secure}");
```

Comment:
Shall we undo formatting change. It is clear when expected message is in one line.